executor/show.go

```diff
@@ -861,7 +861,7 @@ func (e *ShowExec) fetchShowCreateDatabase() error {
 }
 
 func (e *ShowExec) fetchShowCollation() error {
-	collations := charset.GetCollations()
```

Comment:
can we remove this function in parser now?

Reply:
It looks reasonable, I will try to remove it in another PR, thanks